### PR TITLE
View entry feature

### PR DESCRIPTION
**Pull request by KennethDelaCruz**
view entry feature 1

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -32,3 +34,13 @@
     width: 50%;
   }
 }
+
+#entries-title {
+  display: inline;
+}
```
Comment (codingcodymiller) [inline css/layout.css]: This is not necessary, since it is already in a `flex` container. Remember, the default behavior of a flex container is to make all children sit on one line

File: css/styles.css [hunk 2]
```diff
@@ -63,8 +58,36 @@ ul {
   padding: 0;
   display: flex;
   align-items: center;
+  flex-wrap: wrap;
```
Comment (codingcodymiller) [inline css/styles.css]: Why is this necessary?

File: js/data.js [hunk 3]
```diff
@@ -6,10 +6,19 @@ var data = {
...
 if (previousData !== null) {
   data = JSON.parse(previousData);
+  allContainerDiv.forEach(hideDIV);
+}
+
+function hideDIV(object) {
+  var viewType = object.getAttribute('data-view');
+  if (viewType !== data.view) {
+    object.setAttribute('class', 'container hidden');
+  }
```
Comment (codingcodymiller) [inline js/data.js]: You should do this in `main.js`

File: js/data.js [hunk 4]
```diff
@@ -6,10 +6,19 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
-
+var allContainerDiv = document.querySelectorAll('.container');
```
Comment (codingcodymiller) [inline js/data.js]: You should do this in `main.js`

File: js/main.js [hunk 5]
```diff
@@ -15,18 +19,86 @@ function imgPreview(event) {
...
-  event.preventDefault();
   var postInfo = {
     title: formRoute.title.value,
     imageURL: formRoute.img.value,
     notes: formRoute.notes.value,
     entryID: data.nextEntryId
   };
   data.nextEntryId++;
-  data.entries.unshift(postInfo);
+  data.entries.push(postInfo);
```
Comment (codingcodymiller) [inline js/main.js]: This should still be `unshift`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good work so far! please see my notes

**Comment by KennethDelaCruz** [comment]
view entry feature 1

File: index.html [hunk 6]
```diff
@@ -16,20 +16,29 @@
...
-    <div class="container" data-view="entries">
-      <ul>
-        <li>
+    <div class="container" id="entries" data-view="entries">
+      <div class="column-full entry-view-header">
+        <h1 id="entries-title">Entries</h1>
+        <button id="new-post-button">New Post</button>
+      </div>
+      <ul class="post-entries">
+        <li id="example-post">
```
Comment (thebearingedge) [inline index.html]: Please remove the example once the feature is working.

File: js/main.js [hunk 7]
```diff
@@ -1,11 +1,23 @@
...
+var postButton = document.querySelector('#new-post-button');
+var allContainerDiv = document.querySelectorAll('.container');
+var formDiv = document.querySelector('#form-container');
+var entriesDiv = document.querySelector('#entries');
+var examplePost = document.querySelector('#example-post');
+var postList = document.querySelector('#post-list');
+
+if (data.entries.length >= 1) {
+  examplePost.setAttribute('class', 'hidden');
+}
```
Comment (thebearingedge) [inline js/main.js]: You can delete this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice, just a little cleanup left.

**Review by thebearingedge — APPROVED** [review]
